commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
```

Comment:
Can we also include https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-lerna-scopes to lint the scopes?

Reply:
Yep!